http/basic/README.md

```diff
@@ -0,0 +1,38 @@
+# Generic HTTP Protocol Binding
+
+[![Javadocs](http://www.javadoc.io/badge/io.cloudevents/cloudevents-http-basic.svg?color=green)](http://www.javadoc.io/doc/io.cloudevents/cloudevents-http-basic)
+
+This module is designed to be usable with various HTTP APIs.
```

Comment:
@slinkydeveloper How should I improve this?

Reply:
sounds ok to me, I would prefer you add links to other http impls (not only vertx, but also restful-ws)